tests/cupy_tests/lib_tests/test_strided_tricks.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+import numpy as np
+import six
+
+import cupy as cp
+from cupy import cuda
+from cupy import testing
+from cupy.testing import condition
+from cupy.testing.array import assert_array_equal
+from cupy.lib.stride_tricks import as_strided
+
+
+@testing.gpu
+class TestAsStrided(unittest.TestCase):
+    def test_as_strided(self):
+        a = cp.array([1, 2, 3, 4])
+        a_view = as_strided(a, shape=(2,), strides=(2 * a.itemsize,))
+        expected = cp.array([1, 3])
+        assert_array_equal(a_view, expected)
+
+        a = cp.array([1, 2, 3, 4])
+        a_view = as_strided(a, shape=(3, 4), strides=(0, 1 * a.itemsize))
+        expected = cp.array([[1, 2, 3, 4], [1, 2, 3, 4], [1, 2, 3, 4]])
+        assert_array_equal(a_view, expected)
+
+    def test_rolling_window(self):
```

Comment:
Could you use `@testing.numpy_cupy_array_equal()` and `testing.shaped_arange(shape, xp)`?